src/Calculator/Common/AppLifecycleLogger.cpp

```diff
@@ -35,7 +35,7 @@ namespace CalculatorApp
     AppLifecycleLogger::AppLifecycleLogger()
         : m_appLifecycleProvider(
             L"Microsoft.Windows.AppLifeCycle",
-            LoggingChannelOptions(GUID{ 0x4f50731a, 0x89cf, 0x4782, 0xb3, 0xe0, 0xdc, 0xe8, 0xc9, 0x4, 0x76, 0xba }), // Microsoft Telemetry group
+            LoggingChannelOptions(GUID{ 0x4f50731a, 0x89cf, 0x4782, 0xb3, 0xe0, 0xdc, 0xe8, 0xc9, 0x4, 0x76, 0xba }), // Microsoft Diagnostics group
```

Comment:
>Microsoft Diagnostics group [](start = 121, length = 27)

I appreciate why you've made this change elsewhere, but in this particular case the new comment really isn't correct. I think we should revert this one. (Same feedback applies in TraceLogger.cpp)

Reply:
I was asked to remove all instance of "telemetry" to be consistent with the rest of Microsoft use of diagnostics. Should I just remove the comment?

Reply:
Removing the comment would be OK.